setup.py

```diff
@@ -1,143 +1,111 @@
 #!/usr/bin/env python
 
-"""
-============
-Installation
```

Comment:
Moving this will probably break the docs, did you try it out?

This isn't an uncommon convention to put this here, was the motivation to reduce duplication with `README.md`?